mteb/tasks/Retrieval/eng/LEMBNarrativeQARetrieval.py

```diff
@@ -0,0 +1,67 @@
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class LEMBNarrativeQARetrieval(AbsTaskRetrieval):
+    _EVAL_SPLIT = "test"
+
+    metadata = TaskMetadata(
+        name="LEMBNarrativeQARetrieval",
+        dataset={
+            "path": "dwzhu/LongEmbed",
+            "revision": "6e346642246bfb4928c560ee08640dc84d074e8c",
+            "name": "narrativeqa",
+        },
+        reference="https://huggingface.co/datasets/dwzhu/LongEmbed",
+        description=("narrativeqa subset of dwzhu/LongEmbed dataset."),
+        type="Retrieval",
+        category="s2p",
+        eval_splits=[_EVAL_SPLIT],
+        eval_langs=["eng-Latn"],
+        main_score="ndcg_at_10",
+        date=None,
+        form=None,
+        domains=None,
+        task_subtypes=None,
+        license=None,
+        socioeconomic_status=None,
+        annotations_creators=None,
+        dialect=None,
+        text_creation=None,
+        bibtex_citation=None,
+        n_samples=None,
+        avg_character_length=None,
```

Comment:
These need to be filled out. The docstring of the taskmetadata gives you an introduction to each field.

Reply:
ok, I have filled some key parameters. Personally, I'm confused by `socioeconomic_status`, what do you mean by saying the `socioeconomic_status` of a dataset?